storage_file/models/storage_file.py

```diff
@@ -146,9 +146,11 @@ def _compute_url(self):
                     base_url, record._slugify_name_with_id()
                 )
             else:
-                record.url = "{}/{}".format(
-                    record.backend_id.base_url, record.relative_path
-                )
+                parts = [record.backend_id.base_url]
```

Comment:
@simahawk You are here on a point discussed a lot of times with @Cedric-Pigeon. His POV is that the `directory_path` is the internal path used to store you image into the backend. The url should not use this information. If the file is not served at the root of `base_url`, it's your responsibility to put  in place a redirection or configure a CDN (for S3)

If you use a CDN into S3 you can use the `shopivader_backend_image_proxy` to rewrite this URL...  when the info is exported to alglio.... 

I'm not convince by this sapproach and we can discuss about it with @Cedric-Pigeon and @sebastienbeau

Reply:
hmm then we should make it optional because w/ exoscale by default it won't work.
I add a param to include it or not they every one is free to use it or not, fine?